project3/index.html

```diff
@@ -0,0 +1,410 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<link href="https://fonts.googleapis.com/css?family=BenchNine:300,400,700&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css?family=Teko:300,400,500,600,700&display=swap" rel="stylesheet">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>Super Hero registry</title>
+</head>
+
+<body>
+    <?xml version="1.0" encoding="utf-8"?>
+    <svg viewBox="5 4.973 340 338.054" xmlns="http://www.w3.org/2000/svg" class="shield">
+        <ellipse cx="175" cy="174" rx="170" ry="169.027" style="stroke: rgb(255, 255, 255); stroke-width: 2px;" />
+        <path
+            d="M 73.895 65.03 C 64.66 73.591 56.543 83.283 49.737 93.877 L 175 231.997 L 299.789 93.474 C 292.838 82.853 284.558 73.164 275.148 64.646 L 207.21 118.741 L 190.98 84.357 C 190.98 84.357 194.97 72.179 209.098 78.317 C 209.098 78.317 212.785 64.842 190.989 64.842 L 172.316 64.842 C 172.316 64.842 165.766 63.796 160.586 74.685 L 139.711 118.366 L 73.268 66.211"
+            style="stroke-width: 2px; fill: rgb(250, 250, 250);" />
+        <path
+            d="M 86.421 292.803 C 138.253 330.955 208.686 331.651 261.261 294.53 L 203.444 206.846 L 174.812 238.529 L 144.74 206.846 Z M 310.302 235.46 C 318.008 218.748 322.495 200.731 323.526 182.357 L 270.468 135.527 L 246.221 165.491 Z M 323.651 177.722 C 323.651 176.479 323.75 175.244 323.75 174 C 323.788 146.694 316.199 119.92 301.838 96.695 L 272.714 132.717 Z M 243.948 168.301 L 219.504 198.543 L 281.053 277.709 C 292.276 266.359 301.594 253.269 308.647 238.949 Z M 37.899 235.46 C 30.196 218.747 25.712 200.731 24.684 182.357 L 77.733 135.527 L 101.972 165.518 Z M 24.55 177.722 C 24.55 176.479 24.452 175.244 24.452 174 C 24.413 146.694 32.002 119.92 46.364 96.695 L 75.487 132.717 Z M 104.253 168.301 L 128.697 198.543 L 67.148 277.709 C 55.925 266.359 46.607 253.269 39.555 238.949 Z"
+            style="stroke-width: 2px; fill: rgb(250, 250, 250);" />
+    </svg>
+    <h1>super hero resistry</h1>
+    <h2>With great power there must also come—great responsibility</h2>
+    <form action="">
+        <div class="box"></div>
+        <div class="content">
+            <!-- <h3 class="sr-only">Identity</h3> -->
+
+            <h3 class="identity">identity</h3>
+
+            <label for="real-name">Real Name</label>
+            <input id="real-name" type="text" required>
+
+            <label for="alias">Alias<input id="alias" type="text" required>
+            </label>
+
+
+            <label for="email">Email <input id="email" type="text">
+            </label>
```

Comment:
While in class I showed you that the `label` and `input` elements could be related in these two different styles (`input` inside and outside of the `label`), I would expect all three of these sets to use similar markup since they are displayed in a similar way -- that is, there's no reason for them to be different, so they shouldn't.